documentation/docker.md

```diff
@@ -251,7 +251,7 @@ Credentials for `twd-tv-ci` for pulling images
 github_packages_username: NotARealUserName
 github_packages_token: NotRealToken
 ```
-These are used within the Gov.UK PaaS web app and worker app [terraform](../terraform/app/modules/paas/main.tf), like so:
+These are used within the AKS web app and worker app Terraform, like so:
```

Comment:
twd-tv-ci is not used anymore, the section `#### The `twd-tv-ci` user:` should be deleted as we use [the GITHUB_TOKEN instead](https://github.com/DFE-Digital/teaching-vacancies/blob/main/.github/workflows/build_and_deploy.yml#L105). Please link to https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry#authenticating-in-a-github-actions-workflow